Week7/app/src/main/java/com/mmoreno/pokeapp/ui/paging/PokePagedListAdapter.kt

```diff
@@ -1,4 +1,4 @@
-package com.mmoreno.pokeapp
+package com.mmoreno.pokeapp.ui.paging
```

Comment:
This is a neat approach, I learned from that article as well, it's not a pattern I was familiar with but Alex is an excellent developer and I really enjoy his work on the site. Nice job incorporating it into your project.